public/i18n/en/translation.json

```diff
@@ -1661,16 +1661,16 @@
       "base_dn_tooltip": "Base Distinguished Name (DN) that serves as the starting point for LDAP searches. Example: 'dc=secureexample,dc=com' for Active Directory or 'dc=directory,dc=nh' for NethServer OpenLDAP",
       "user_dn": "User DN",
       "user_dn_tooltip": "Distinguished Name (DN) for the user in the LDAP directory. The DN uniquely identifies the user's location within the LDAP tree structure. Example: 'cn=Users,dc=secureexample,dc=com' for Active Directory or 'ou=People,dc=directory,dc=nh' for NethServer OpenLDAP",
-      "user_cn": "User CN field",
-      "user_cn_tooltip": "Common Name (CN) for the user. The CN is a component of the user's DN and typically represents the user's full name or another identifier. Example: 'cn'",
+      "user_display_attribute": "User display name field",
+      "user_display_attribute_tooltip": "Display name attribute for the user. The display name typically represents the user's full name like 'John Doe'. Suggestion: use 'displayName' for Active Directory or 'cn' for NethServer OpenLDAP",
       "bind_dn": "Bind DN",
       "bind_dn_tooltip": "Distinguished Name (DN) used to bind to the LDAP directory. The DN is used to authenticate the connection to the LDAP server. Example: 'cn=Administrator,cn=Users,dc=secureexample,dc=com' for Active Directory or 'cn=ldapservice,dc=directory,dc=nh' for NethServer OpenLDAP",
       "bind_password": "Bind password",
       "bind_password_tooltip": "Password for Bind DN, used to authenticate the connection to the LDAP server",
       "user_bind_dn": "Custom user bind DN",
-      "user_bind_dn_tooltip": "Distinguished Name (DN) used to bind to the LDAP directory. This overrides the calculated user bind DN. Use it when authenticating against an Active Directory. It can contain a '%u' variable that will be expanded to the user name. Example: '%u{'@'}secretexample.com', 'SECRETEXAMPLE\\%u'",
+      "user_bind_dn_tooltip": "Distinguished Name (DN) used to bind to the LDAP directory. This overrides the calculated user bind DN. Use it when authenticating against an Active Directory. It can contain a '%u' variable that will be expanded to the user name. Suggestion: use '%u{'@'}secretexample.com' and leave blank for NethServer OpenLDAP",
```

Comment:
```suggestion
      "user_bind_dn_tooltip": "Distinguished Name (DN) used to bind to the LDAP directory. This overrides the calculated user bind DN. Use it when authenticating against an Active Directory. It can contain a '%u' variable that will be expanded to the user name. Suggestion: use '%u{'@'}secretexample.com' or leave blank for NethServer OpenLDAP",
```

Reply:
Thanks for pointing it out. Done.